pebble_tool/util/static/js/textdecoder.js

```diff
@@ -0,0 +1,672 @@
+// Copyright 2014 Joshua Bell. All rights reserved.
```

Comment:
This entire file could probably be omitted if you remove support for packing strings from "pack" — which I'm pretty sure you aren't using.